User folder has spaces and cibuild.cmd fails

My .nuget folder is at `C:\Users\Miguel Angelo\.nuget\`

When running `cibuild.cmd` I get this message:

```
EXEC : error : Angelo\.nuget\packages\....
```

Note that the User folder is split in half. May that be the problem?

The full error screen:

```
  Restoring build tools...
  log  : Restoring packages for C:\Projetos\msbuild\src\.nuget\project.json...
  log  : Installing NuGet.Packaging.Core 4.5.0-preview2-4529.
  log  : Installing NuGet.Packaging 4.5.0-preview2-4529.
  log  : Installing System.Runtime.InteropServices 4.3.0.
  log  : Installing NuGet.Versioning 4.5.0-preview2-4529.
  log  : Installing NuGet.Protocol 4.5.0-preview2-4529.
  log  : Installing NuGet.LibraryModel 4.5.0-preview2-4529.
  log  : Installing NuGet.Common 4.5.0-preview2-4529.
  log  : Installing Newtonsoft.Json 9.0.1.
  log  : Installing NuGet.DependencyResolver.Core 4.5.0-preview2-4529.
  log  : Installing xunit.runner.console 2.1.0.
  log  : Installing NuGet.Configuration 4.5.0-preview2-4529.
  log  : Installing NuGet.ProjectModel 4.5.0-preview2-4529.
  log  : Installing NuGet.Frameworks 4.5.0-preview2-4529.
  log  : Installing NuGet.Commands 4.5.0-preview2-4529.
  log  : Installing Microsoft.DotNet.Build.Tasks.Feed 2.1.0-prerelease-02221-02.
  log  : Installing MicroBuild.Core 0.2.0.
  log  : Installing NuGet.Build.Tasks 4.5.0-preview2-4529.
  log  : Installing NuSpec.ReferenceGenerator 1.4.2.
  log  : Installing Microsoft.DotNet.BuildTools.GenAPI 1.0.0-beta2-00731-01.
  log  : Installing Microsoft.Net.Compilers 2.3.1.
  log  : Installing Nerdbank.GitVersioning 1.6.35.
  log  : Writing lock file to disk. Path: C:\Projetos\msbuild\src\.nuget\project.lock.json
  log  : Generating MSBuild file C:\Projetos\msbuild\src\.nuget\.nuget.nuget.targets.
  log  : Generating MSBuild file C:\Projetos\msbuild\src\.nuget\.nuget.nuget.props.
  log  : Restore completed in 110997ms for C:\Projetos\msbuild\src\.nuget\project.json.
  Building Project(s): build\dirs.proj : Configuration=Debug, Platform=x86
  Building Project(s): src\dirs.proj : Configuration=Debug, Platform=x86
  Building Project(s): Samples\dirs.proj : Configuration=Debug, Platform=x86
EXEC : error : Angelo\.nuget\packages [C:\Projetos\msbuild\src\Framework\Microsoft.Build.Framework.csproj]
C:\Projetos\msbuild\Tools\packageresolve.targets(35,5): error MSB3073: O comando ""C:\Projetos\msbuild\Tools\dotnetcli/
dotnet.exe" restore --legacy-packages-directory --packages C:\Users\Miguel Angelo\.nuget\packages "C:\Projetos\msbuild\
src\Framework/project.json"" foi encerrado com o código 1. [C:\Projetos\msbuild\src\Framework\Microsoft.Build.Framework
.csproj]
EXEC : error : Angelo\.nuget\packages [C:\Projetos\msbuild\src\MSBuildTaskHost\MSBuildTaskHost.csproj]
C:\Projetos\msbuild\Tools\packageresolve.targets(35,5): error MSB3073: O comando ""C:\Projetos\msbuild\Tools\dotnetcli/
dotnet.exe" restore --legacy-packages-directory --packages C:\Users\Miguel Angelo\.nuget\packages "C:\Projetos\msbuild\
src\MSBuildTaskHost/project.json"" foi encerrado com o código 1. [C:\Projetos\msbuild\src\MSBuildTaskHost\MSBuildTaskHo
st.csproj]
EXEC : error : Angelo\.nuget\packages [C:\Projetos\msbuild\Samples\TaskUsageLogger\TaskUsageLogger.csproj]
EXEC : error : Angelo\.nuget\packages [C:\Projetos\msbuild\Samples\Dependency\Dependency.csproj]
C:\Projetos\msbuild\Tools\packageresolve.targets(35,5): error MSB3073: O comando ""C:\Projetos\msbuild\Tools\dotnetcli/
dotnet.exe" restore --legacy-packages-directory --packages C:\Users\Miguel Angelo\.nuget\packages "C:\Projetos\msbuild\
Samples\TaskUsageLogger/project.json"" foi encerrado com o código 1. [C:\Projetos\msbuild\Samples\TaskUsageLogger\TaskU
sageLogger.csproj]
EXEC : error : Angelo\.nuget\packages [C:\Projetos\msbuild\Samples\XmlFileLogger\XmlFileLogger.csproj]
C:\Projetos\msbuild\Tools\packageresolve.targets(35,5): error MSB3073: O comando ""C:\Projetos\msbuild\Tools\dotnetcli/
dotnet.exe" restore --legacy-packages-directory --packages C:\Users\Miguel Angelo\.nuget\packages "C:\Projetos\msbuild\
Samples\Dependency/project.json"" foi encerrado com o código 1. [C:\Projetos\msbuild\Samples\Dependency\Dependency.cspr
oj]
C:\Projetos\msbuild\Tools\packageresolve.targets(35,5): error MSB3073: O comando ""C:\Projetos\msbuild\Tools\dotnetcli/
dotnet.exe" restore --legacy-packages-directory --packages C:\Users\Miguel Angelo\.nuget\packages "C:\Projetos\msbuild\
Samples\XmlFileLogger/project.json"" foi encerrado com o código 1. [C:\Projetos\msbuild\Samples\XmlFileLogger\XmlFileLo
gger.csproj]


** Build completed. Exit code: 1

Bootstrap build failed with errorlevel 1

---------------------------------------
- cibuild.cmd FAILED. -
---------------------------------------
```

--- a/dir.props
+++ b/dir.props
@@ -89,7 +89,7 @@
     <DnuCoreRestoreCommand>$(DnuCoreRestoreCommand) --legacy-packages-directory</DnuCoreRestoreCommand>
 
     <!-- packageresolve.targets from build tools uses the DnuRestoreCommand property. Setting it to use dotnet CLI under the hood-->
-    <DnuRestoreCommand>$(DnuCoreRestoreCommand) --packages $(PackagesDir)</DnuRestoreCommand>
+    <DnuRestoreCommand>$(DnuCoreRestoreCommand) --packages &quot;$(PackagesDir)&quot;</DnuRestoreCommand>
   </PropertyGroup>
 
   <!-- Common nuget properties -->

